src/plugins/dump/dump.cpp

```diff
@@ -524,18 +524,18 @@ class FileStreamBuf : public std::streambuf
 	char buf_;
 };
 
-int freadks (KeySet * ks, FILE * file, Key * errorKey)
+int funserialize (KeySet * ks, FILE * file, Key * errorKey)
 {
 	FileStreamBuf buf (file);
 	std::istream is (&buf);
-	return unserialise (is, errorKey, ks, true);
+	return unserialize (is, errorKey, ks, true);
 }
 
-int fwriteks (KeySet * ks, FILE * file, Key * errorKey)
+int fserialize (KeySet * ks, FILE * file, Key * errorKey)
 {
 	FileStreamBuf buf (file);
 	std::ostream os (&buf);
-	return serialise (os, errorKey, ks, true);
+	return serialize (os, errorKey, ks, true);
```

Comment:
Please add this change also in release notes. Also write a bit more about how to use Java plugins there. Can you also add

Reply:
I think part of the last sentence is missing...

Reply:
Yes, it was also about something in the release notes.

Reply:
As this is hopefully the last PR before the release, please see what you can do to improve the release notes.

Reply:
I have already added some stuff (Highlights, dump)